deploy/template.yaml

```diff
@@ -3520,7 +3520,9 @@ Resources:
     Properties:
       ActionsEnabled: true
       AlarmActions:
-        - !ImportValue platform-alarm-topic-slack-warning-alert
```

Comment:
Is this intentionally different from the other CRIs?

Reply:
Wrong value copied, has been updated now.